test/mock_windows.uts

```diff
@@ -0,0 +1,70 @@
+% Regression tests on Windows only for Scapy
+
+# More informations at http://www.secdev.org/projects/UTscapy/
+# $Id: regression.uts,v 1.9 2008/07/29 15:30:29 pbi Exp pbi $
+
+############
+############
```

Comment:
You need a title starting with "+". That's why there is an error with the appveror tests.